superset/assets/cypress/integration/explore/visualization_tests.js

```diff
@@ -4,7 +4,6 @@
 
```

Comment:
Perhaps break this file into multiple files, one per chart, to avoid this becoming `viz.py`